introduction.md

```diff
@@ -1,2 +1,8 @@
 My name is Bob. I like pina coladas and walks in the rain.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?